_posts/2015-07-23-postgres-review-operators.md

```diff
@@ -0,0 +1,292 @@
+---
+layout:     post
+title:      Review of operators in Postgres
+date:       2015-07-23 23:00:00
+summary:    In this post we will review some operators so we can have
```

Comment:
If this summary is showed in someplace I believe it needs change. The user doesn't know the next post, so the post needs to be unique. What will I learn?